snuba/query/snql/parser.py

```diff
@@ -1297,11 +1297,11 @@ def _align_max_days_date_align(
     )
     if not lower:
         raise ParsingException(
-            f"missing >= condition on column {entity.required_time_column} for entity {key.value}"
+            f"missing >= condition with a datetime literal on column {entity.required_time_column} for entity {key.value}"
```

Comment:
we could probably put an example here of what a "datetime literal" looks like and make this error message easier to understand
```suggestion
            f"""missing >= condition with a datetime literal on column {entity.required_time_column} for entity {key.value}
            
            Example:
                {entity.required_time_column} >=  toDateTime('{datetime.now().replace(microsecond=0).isoformat()}')
                AND {entity.required_time_column} <  toDateTime('{datetime.now().replace(microsecond=0).isoformat()}')
            """
```

Reply:
It seems there are some issues with multi-line error messages, but I agree that an example would make it easier to understand. Maybe, we could condense it into a single line.
![image](https://github.com/getsentry/snuba/assets/132949946/f2060457-7331-459d-ad34-9678b685e2e9)

Reply:
that would work